llama_finetuning.py

```diff
@@ -127,14 +151,16 @@ def main(**kwargs):
 
         mixed_precision_policy, wrapping_policy = get_policies(fsdp_config, rank)
         my_auto_wrapping_policy = fsdp_auto_wrap_policy(model, LlamaDecoderLayer)
-   
+
         model = FSDP(
             model,
             auto_wrap_policy= my_auto_wrapping_policy if train_config.use_peft else wrapping_policy,
             mixed_precision=mixed_precision_policy if not fsdp_config.pure_bf16 else None,
             sharding_strategy=fsdp_config.sharding_strategy,
             device_id=torch.cuda.current_device(),
             limit_all_gathers=True,
+            sync_module_states=True,
```

Comment:
@lchu-ibm shall we add the check here as well, in case low_cpu and nightlies use this else None.

Reply:
https://github.com/facebookresearch/llama-recipes/pull/77/commits/c19c5c69aa4f2b0eb03119ef64c3aec164595ab9